README.md

```diff
@@ -57,6 +57,9 @@
 
 *(full video can be found [here](https://drive.google.com/file/d/1G5OaCsPco_4H7F5-s6n2Mm3wI5V9K6WE/view?usp=sharing))*
 
+<p align="center">
+    <img src="https://raw.githubusercontent.com/TwentyBN/sense/reactive-gesture-demo-pr/docs/gifs/hand_gesture_event_detection.gif" width="600px" height="300px">
```

Comment:
@corneliusboehm  I've added a GIF to the Readme, as per your suggestion. The link will have to be changed when we merge: `reactive-gesture-demo-pr` >> `master`

Reply:
I used relative links now instead which should work on both feature and master branches.